stacks/apps/cms.yml

```diff
@@ -158,28 +158,34 @@ Resources:
               - Action: sns:Publish
                 Effect: Allow
                 Resource: !Ref PorterJobExecutionSnsTopicArn
+                Sid: AllowPublish
             Version: "2012-10-17"
-          PolicyName: PorterSnsPublish
+          PolicyName: Porter
         - PolicyDocument:
             Statement:
-              - Action: sqs:*
+              - Action:
+                  - sqs:ChangeMessageVisibility
+                  - sqs:DeleteMessage
+                  - sqs:GetQueueAttributes
+                  - sqs:GetQueueUrl
+                  - sqs:ReceiveMessage
+                  - sqs:SendMessage
                 Effect: Allow
                 Resource:
                   - !GetAtt AudioCallbackQueue.Arn
                   - !GetAtt ImageCallbackQueue.Arn
                   - !GetAtt PodcastImportQueue.Arn
                   - !GetAtt SearchIndexerQueue.Arn
                   - !GetAtt DefaultJobQueue.Arn
+                Sid: AllowShoryuken
             Version: "2012-10-17"
-          PolicyName: JobQueues
+          PolicyName: AppQueueus
         - PolicyDocument:
             Statement:
-              - Action: sqs:* # TODO Seems very permissive
-                Effect: Allow
-                Resource: !Sub arn:${AWS::Partition}:sqs:${AWS::Region}:${AWS::AccountId}:${AnnounceResourcePrefix}*
-              - Action: sns:* # TODO Seems very permissive
+              - Action: sns:Publish
```

Comment:
I believe in the new CFN-friendly mode, all we need for SNS is `publish`

Reply:
Is that true?  It's running [rake announce:verify_config](https://github.com/PRX/announce/blob/master/lib/tasks/announce.rake#L9).  Which ... hrmm ... just call these [verify](https://github.com/PRX/announce/blob/29f6b13b694719c35f5206e7815826dfc551bbf3/lib/announce/adapters/shoryuken_adapter.rb#L102) methods.  That just log stuff, instead of making any attempt to verify them?

Okay, so maybe this is fine.

Reply:
That was my reading of it as well, just logging